code/modules/library/barcode_scanner.dm

```diff
@@ -8,9 +8,8 @@
 	w_class = WEIGHT_CLASS_TINY
 	///Weakref to the library computer we are connected to.
 	var/datum/weakref/computer_ref
-	///The scan mode we are currently on.
-	///Selections: BARCODE_SCANNER_CHECKIN: check in books, BARCODE_SCANNER_INVENTORY: add scanned books to inventory.
-	var/scan_mode = BARCODE_SCANNER_CHECKIN
+	///Boolean on what mode we're scanning. TRUE is inventory mode, FALSE is check-in mode.
+	var/scan_mode = FALSE
```

Comment:
I should have been more clear, I think defines are useful for this, but I don't think you should rely on it being a boolean for your toggle check. should just use an if else instead.

Reply:
I originally made it use defines and wanted to use that instead, but when I had cut it down to only 2 modes I thought it might as well have just been a bool for simplicity

Reply:
I think defines are better here, a bool "scan mode" is pretty unclear